cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -1040,6 +1040,19 @@ def test__validate_cell_type_ontology_term_id_error(
             in validator.errors[0]
         )
 
+    def test__validate_embeddings_non_nans(self):
+        validator: Validator = Validator()
+        validator._set_schema_def()
+        validator.adata = adata_visium.copy()
+        validator.visium_and_is_single_true_matrix_size = 2
+
+        # invalidate spatial embeddings with NaN value
+        validator.adata.obsm["spatial"][0, 1] = np.nan
+        # Confirm spatial is valid.
+        validator.validate_adata()
+        assert len(validator.errors) == 1
```

Comment:
nit: you could accomplish this in just one assert
```
assert validator == ["ERROR: adata.obs['spatial] contains at least one NaN value."]
```

Reply:
yep, absolutely. done.